[2023-04-01 13:48:29.305 +05:30  INF]  9015831752
[2023-04-01 13:49:11.773 +05:30  INF]  
[2023-04-01 14:34:51.391 +05:30  INF]  Data Source=192.168.1.33;Initial Catalog=AVVNL_CALL_CENTER_TEST;Trusted_Connection=False;uid=crmuser;password=Admin@1234;
[2023-04-01 14:35:00.424 +05:30  INF]  Login success
[2023-04-01 14:42:10.459 +05:30  INF]  Data Source=192.168.1.33;Initial Catalog=AVVNL_CALL_CENTER_TEST;Trusted_Connection=False;uid=crmuser;password=Admin@1234;
[2023-04-01 14:42:17.700 +05:30  INF]  Invalid credentials
[2023-04-01 14:43:19.278 +05:30  INF]  Data Source=192.168.1.33;Initial Catalog=AVVNL_CALL_CENTER_TEST;Trusted_Connection=False;uid=crmuser;password=Admin@1234;
[2023-04-01 14:43:28.806 +05:30  INF]  Login success
[2023-04-01 14:46:23.099 +05:30  INF]  8769576997
[2023-04-01 14:46:54.431 +05:30  INF]  
[2023-04-01 14:49:15.787 +05:30  ERR]  An unhandled exception has occurred while executing the request.
System.Data.SqlClient.SqlException (0x80131904): Error converting data type varchar to bigint.
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   at System.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   at System.Data.SqlClient.SqlDataReader.Read()
   at System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   at System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   at System.Data.Common.DataAdapter.Fill(DataSet dataSet, String srcTable, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   at System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   at System.Data.Common.DbDataAdapter.Fill(DataSet dataSet, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   at System.Data.Common.DbDataAdapter.Fill(DataSet dataSet)
   at CallCenterCoreAPI.Database.Repository.SqlHelper.ExecuteDataset(SqlConnection connection, CommandType commandType, String commandText, SqlParameter[] commandParameters) in D:\git\CallCenterAPI\CallCenterCoreAPI\Database\SqlHelper\SqlHelper.cs:line 872
   at CallCenterCoreAPI.Database.Repository.SqlHelper.ExecuteDataset(String connectionString, CommandType commandType, String commandText, SqlParameter[] commandParameters) in D:\git\CallCenterAPI\CallCenterCoreAPI\Database\SqlHelper\SqlHelper.cs:line 787
   at CallCenterCoreAPI.Database.Repository.ComplaintRepository.GetPreviousComplaintNo(String complaintNo) in D:\git\CallCenterAPI\CallCenterCoreAPI\Database\Repository\ComplaintRepository.cs:line 181
   at CallCenterCoreAPI.Controllers.ComplaintController.SearchComplaintByComplaintNo(ComplaintSearchQueryModel complaintSearchQueryModel) in D:\git\CallCenterAPI\CallCenterCoreAPI\Controllers\ComplaintController.cs:line 67
   at lambda_method21(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
ClientConnectionId:e4bbf712-7e26-40ad-8749-6b56a0b44f8f
Error Number:8114,State:5,Class:16
